api/models.py

```diff
@@ -247,7 +265,37 @@ def translate_fields(cls, params: dict):
         parent = params.get('parent')
         if parent:
             translated['parent'] = ObjectId(parent)
-        return translated
+
+        timestamp_fields = ('created', 'updated', 'timeout', 'holdoff')
+
+        operator_map = {
```

Comment:
I wonder if we could make this more generic to allow any field to be compared with a value rather than just timestamps.

Reply:
Also these are MongoDB operators, but the models defined in this module shouldn't have any knowledge of the database engine being used.  So this operator translation should be done in `Database` instead.

Reply:
I agree, the operator translation should be in DB layer.
I am working on it.

Reply:
I updated operator mapping to be more generic to apply it to all attributes rather than just timestamp fields.